### PR TITLE
feat: Enable removal of legacy passkeys

### PR DESCRIPTION
**Pull request by sea-snake**
Enable users to remove legacy passkeys when their recovery phrase has been activated and verified. 

# Changes

- Add disabled and tooltip item options to `<Select/>` component.
- Show the select dropdown for legacy passkeys and disable the remove button with a message unless a recovery phrase has been activated and verified. 
- Add `Legacy` label to passkeys to indicate they're legacy passkeys, additionally update their description to clarify they're no longer usable and should be removed.

# Tests

- Added `can remove a legacy passkey` test cases.

### CODE REVIEW

File: src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts [hunk 1]
```diff
@@ -151,6 +155,103 @@ test.describe("Access methods", () => {
...
+
+    test("unless recovery phrase is missing or unverified", async ({
+      manageAccessPage,
+    }) => {
+      // Assert remove button is disabled
+      await manageAccessPage.assertPasskeyCount(2);
+      await manageAccessPage
+        .findPasskey(LEGACY_PASSKEY_NAME)
+        .assertUnremovable();
+    });
```
[the remark this answers is not in the document]
Comment (sea-snake) [reply src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts:254]: Good catch!

File: src/frontend/src/routes/(new-styling)/manage/(authenticated)/(access-and-recovery)/access/+page.svelte [hunk 2]
```diff
@@ -64,6 +61,20 @@
...
+      const value = data.identityInfo.authn_methods.find(
+        (m) =>
+          "Recovery" in m.security_settings.purpose &&
+          getMetadataString(m.metadata, "usage") === "recovery_phrase",
+      );
+      return value === undefined
+        ? "missing"
+        : value.last_authentication[0] === undefined
+          ? "unverified"
+          : "verified";
```
[the remark this answers is not in the document]
Comment (sea-snake) [reply src/frontend/src/routes/(new-styling)/manage/(authenticated)/(access-and-recovery)/access/+page.svelte]: Good catch regarding the locked recovery phrase edge case.

File: src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts [hunk 3]
```diff
@@ -151,6 +155,105 @@ test.describe("Access methods", () => {
...
+      await manageAccessPage
+        .findPasskey(LEGACY_PASSKEY_NAME)
+        .remove((dialog) => dialog.confirm());
+
+      // Assert other passkeys are still present
+      await manageAccessPage.assertPasskeyCount(1);
+      await manageAccessPage.assertPasskeyExists("post-upgrade-passkey");
+    });
+
+    test("unless recovery phrase is missing or unverified", async ({
```
Comment (aterga) [inline src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts:246]: These two scenarios could be merged 

1. Before we have a recovery phrase, legacy passkey shouldn't be removable
2. Then we add a recovery phrase and the passkey is not removable

Comment (sea-snake) [reply src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts:246]: Good point!

Comment (aterga) [reply src/frontend/tests/e2e-playwright/routes/manage/access.spec.ts:246]: Can be done later if need be
